integration/hive/src/main/java/org/apache/carbondata/hive/MapredCarbonInputFormat.java

```diff
@@ -213,6 +223,45 @@ private QueryModel getQueryModel(Configuration configuration, String path)
         .build();
   }
 
+  private static CarbonTable getCarbonTable(Configuration tableProperties)
```

Comment:
Please move this method to HiveCarbonUtil class.

Reply:
done